### PR TITLE
Groq benchmarking support

### PR DESCRIPTION
**Pull request by danielholanda**
**Description:** This PR adds Groq benchmarking support to mlagility. 

**Testing:** Ensure that you the Groq SDK installed locally, as well as a remote server with the Groq SDK and a miniconda environment named 'groqflow' installed. Then, simply run `benchit any_script.py --device groq`.

### CODE REVIEW

File: src/mlagility/api/cloud.py [hunk 1]
```diff
@@ -145,6 +158,24 @@ def configure_remote(accelerator: str) -> Tuple[str, str]:
...
 
+def setup_groqchip_host(client) -> None:
+    # Make sure at least one GroqChip is available remotely
+    stdout, exit_code = exec_command(client, "/usr/bin/lspci -n")
+    if stdout == "" or exit_code == 1:
+        msg = "Failed to run lspci to get GroqChips available"
+        raise exp.GroqModelRuntimeError(msg)
+    num_chips_available = sdk.get_num_chips_available(stdout.split("\n"))
+    if num_chips_available < 1:
+        raise exp.GroqModelRuntimeError("No GroqChips found")
```
Comment (jeremyfowers) [inline src/mlagility/api/cloud.py:169]: brand: GroqChip Processors

File: src/mlagility/api/cloud.py [hunk 2]
```diff
@@ -145,6 +158,24 @@ def configure_remote(accelerator: str) -> Tuple[str, str]:
...
+    print(f"{num_chips_available} GroqChips found")
```
Comment (jeremyfowers) [inline src/mlagility/api/cloud.py:170]: brand: GroqChip Processors

**Review by jeremyfowers — APPROVED** [review]

File: src/mlagility/api/cloud.py (hunk 1 above)
Comment (danielholanda) [reply src/mlagility/api/cloud.py]: solved!

File: src/mlagility/api/cloud.py (hunk 2 above)
Comment (danielholanda) [reply src/mlagility/api/cloud.py]: solved!

**Pull request merged**